samples/apps/txregulator/clients/loader.py

```diff
@@ -46,23 +47,15 @@ def run(args):
 
         # Give regulators permissions to register regulators and banks
         for regulator in regulators:
-            proposal_result, error = network.consortium.propose(
-                0,
-                primary,
-                f"""
-                return Calls:call(
-                    "set_user_data",
-                    {{
-                        user_id = {regulator.ccf_id},
-                        user_data = {{
-                            privileges = {{
-                                REGISTER_REGULATORS = true,
-                                REGISTER_BANKS = true,
-                            }}
-                        }}
-                    }}
-                )
-                """,
+            proposal_body, _ = Proposals.set_user_data_proposal(
```

Comment:
This appears to be untested by the CI so has likely rotted. I've tried to update it appropriately here, but eventually we may need either an automated test or to remove this.

Reply:
I think it's probably time to let it go and replace it with the price comparison sample suggested by Brent, written in Javascript.